controllers/watcher_common.go

```diff
@@ -55,6 +55,9 @@ const (
 	// WatcherApplierLabelPrefix - a unique, service binary specific prefix for the
 	// labels the WatcherApplier controller uses on children objects
 	WatcherApplierLabelPrefix = "watcher-applier"
+	// WatcherDecisionEngineLabelPrefix - a unique, service binary specific prefix
+	// for the labels the WatcherDecisionEngine controller uses on child objects
+	WatcherDecisionEngineLabelPrefix = "watcher-decisionengine"
```

Comment:
I know I'm being picky but, I think this should be watcher-decision-engine :) (that's what watcher uses upstream to identify itself).